CMakeLists.txt

```diff
@@ -23,6 +23,15 @@ option(BUILD_OPENSSL_PLATFORM "If buildng OpenSSL what is the target platform" O
 option(LOCAL_OPENSSL_BUILD "Whether or not to use local OpenSSL build" OFF)
 option(CONSTRAINED_DEVICE "Change pthread stack size" OFF)
 
+# Endpoint selection - mutually exclusive
```

Comment:
Should one of these be ON, as they are mutually exclusive?

Reply:
At most 1 can be on. 0 is acceptable